damus/Views/PostView.swift

```diff
@@ -813,3 +855,10 @@ func build_post(state: DamusState, post: NSMutableAttributedString, action: Post
     return NostrPost(content: content, kind: .text, tags: tags)
 }
 
+
+// Utility function to check if the URL is a video
+func isVideo(url: URL?) -> Bool {
+    guard let url = url else { return false }
+    let videoExtensions = ["mp4", "mov", "avi"]
```

Comment:
We should check for the supported video types by `AVPlayer` and compare instead of using a hard-coded list. Or at least we should research the formats supported by the player and add them here.

Probably using something like https://developer.apple.com/documentation/avfoundation/avurlasset/1386800-audiovisualtypes and https://developer.apple.com/documentation/uniformtypeidentifiers/uttype-swift.struct/init(filenameextension:conformingto:) (The links are suggestions, not necessarily the best answer)

Reply:
Thank you. I researched from the links you have provided and have made the changes.